sources

```diff
@@ -0,0 +1 @@
+08a1420baec329f700b28643790b7781  yarn-offline.tar
```

Comment:
Is this an MD5 hash? Is this for master or release-3.11? (We added some new dependencies over the weekend, so dependencies for the master and release-3.11 branches have already drifted.)

Reply:
Hash of the file, yes (not sure of algorithm). This identifies the file in the dist-git lookaside cache. It's for release-3.11. That's good to know, we'll have to update it right away then.